crates/uv-cli/src/lib.rs

```diff
@@ -4672,6 +4672,17 @@ pub struct PythonPinArgs {
     /// `requires-python` constraint.
     #[arg(long, alias = "no-workspace")]
     pub no_project: bool,
+
+    /// Writes the pinned Python version to a .python-version file in the `XDG_CONFIG_HOME` instead of
+    /// the working directory.
```

Comment:
We try to keep the first line short, because it renders in the short help. We should cover details like `XDG_CONFIG_HOME` in a subsequent line (that's why my suggestion was written that way).

We might want to say "in the uv user configuration directory" then subsequently explain that's `XDG_CONFIG_HOME/uv` and whatever the Windows analogue is.

Reply:
Updated.